### ISSUE TITLE
Reconnection never reconnects

### ISSUE THREAD

**Issue by torhve** [rawdigits/wee-slack#206]
The server log buffer says:

```
01:40	<SYSTEM>	Disconnected from slack, trying to reconnect..
```
And that was 5 days ago according to Glowing Bear date change  marker line:
00:00	<─>	torsdag (31. mars, 5 days later)

**Comment by torhve** [comment]
I think maybe script got stuck during an upgrade of weechat or something along those lines.

**Comment by rawdigits** [comment]
This was (finally) fixed in #257
